docs/configuration/reference/rules.md

```diff
@@ -38,6 +38,14 @@ The `rules` block can be used at the root of a configuration file, or inside an
 
 ---
 
+- message
+- string
+- Optional custom message for this rule.
+  Example: `My Error Derscription. {{message}}`.
+  The {{message}} placeholder (if provided) gets replaced with the actual message of the rule.
```

Comment:
I think this has to be rephrased. cc @Redocly/technical-writers

Reply:
What is the "actual" message of the rule?

Reply:
Each rule, when violated, produces an error message. That is what I called the "actual" one. The `message` field allow users to replace the "actual" message with a custom one.